src/aws/osml/model_runner/inference/http_detector.py

```diff
@@ -170,9 +169,7 @@ def find_features(self, payload: BufferedReader, metrics: MetricsLogger) -> Feat
                 )
             )
             logger.exception(err)
-
-        # Return an empty feature collection if the process errored out
-        return FeatureCollection([])
```

Comment:
If we are not returning an empty collection, I would suggest updating the method return types and docstring as well.

I feel like we should return an emptycollection.

Reply:
It does return a feature collection if the model returns one.